guides-and-tutorials/running-a-signer/how-to-monitor-signer.md

```diff
@@ -0,0 +1,165 @@
+# How to Monitor a Signer
+
+We will use [Grafana Cloud](https://grafana.com) to observe and monitor both the
+Signer and its corresponding Stacks node.
+
+## Requirements
+
+Grafana's application observability docs have a [great
+quick-start](https://grafana.com/docs/grafana-cloud/monitor-applications/application-observability/). We will use:
+
+- Grafana Cloud to collect metrics and visualize them.
+- Grafana Alloy, on the Signer host, to push the metrics.
+
+### Creating a Grafana Cloud account
+
+Before we begin, create a [Grafana
+Cloud](https://grafana.com/docs/grafana-cloud/monitor-applications/application-observability/grafana-cloud/) account (they offer a free tier that you can use).
+
+Once done, access your dashboard, click on "Connections", "Add new connection",
+and select "Hosted Prometheus metrics". Select "Via Grafana Alloy", then on step
+2 select "Run Grafana Alloy" to generate an API token. Note the token
```

Comment:
I think this needs a qualifier - if it's not a gcloud hosted service, will the key ids may be different?
```
By default, grafana will create a hosted instance in gcloud (configurable) - Note the token...
```

Reply:
Well if using a Prometheus instance not hosted by Grafana, users would need to provide their username/password (or token). But I think that's out of scope for this guide, I want to make it as easy as possible.

Reply:
yes, but i think (going off of memory) when you first setup the instance, they give you some options. gcloud being one, aws being another., etc 

in your doc,  the keys are specific to the gcloud option but i would suspect that if someone chose aws, the key id's may be different.